lib/ui/view/channel/channel_view/widgets/channel_chat.dart

```diff
@@ -19,8 +22,108 @@ class ChannelChat extends ViewModelWidget<ChannelPageViewModel> {
               physics: const NeverScrollableScrollPhysics(),
               shrinkWrap: true,
               itemCount: viewModel.channelUserMessages!.length,
-              itemBuilder: (context, index) => ThreadCardView.threadChannelMain(
-                  viewModel.channelUserMessages![index]),
+              itemBuilder: (context, index) => InkWell(
+                onTap: () {
+                  log.i('Tap');
+                },
+                onLongPress: () {
+                  showModalBottomSheet(
```

Comment:
Please use a bottom sheet service for this.
Also, I think one was already created for this